docs/README.md

```diff
@@ -129,95 +176,14 @@ virtualenv -p /usr/bin/python3 .
 ```
 
 ## Mac OS X
-
-In OSX environment the intercept-build tool from
-[scan-build](https://github.com/rizsotto/scan-build) is used to log the
-compiler invocations.
-
-It is possible that the [intercept-build can not
-log](https://github.com/rizsotto/scan-build#limitations)
-the compiler calls without turning off *System Integrity Protection (SIP)*.
-`intercept build` can automatically detect if SIP is turned off.
-
-You can turn off SIP on El Capitan this way:
-
-  * Click the  (Apple) menu.
-  * Select Restart...
-  * Hold down command-R to boot into the Recovery System.
-  * Click the Utilities menu and select Terminal.
-  * Type csrutil disable and press return.
-  * Close the Terminal app.
-  * Click the  (Apple) menu and select Restart....
-
-The following commands are used to bootstrap CodeChecker on
-OS X El Capitan 10.11, macOS Sierra 10.12 and macOS High Sierra 10.13.
-
-```sh
-# Download and install dependencies.
-brew update
-brew install doxygen gcc git
-pip3 install virtualenv
-
-# Install the latest clang see: https://formulae.brew.sh/formula/llvm
-brew install llvm@7
-
-# Fetch source code.
-git clone https://github.com/Ericsson/CodeChecker.git --depth 1 ~/codechecker
-cd ~/codechecker
-
-# Create a Python virtualenv and set it as your environment.
-make venv_osx
-source $PWD/venv/bin/activate
-
-# Build and install a CodeChecker package.
-make package
-
-# For ease of access, add the build directory to PATH.
-export PATH="$PWD/build/CodeChecker/bin:$PATH"
-
-cd ..
-```
-
+For installation instructions for Mac OS X see [Mac OS X Installation Guide](installation_macosx.md) documentation. 
 
 ## Docker
 To run CodeChecker server in Docker see the [Docker](web/docker.md) documentation.
 
 [![Docker](images/docker.jpg)](https://hub.docker.com/r/codechecker/codechecker-web)
```

Comment:
I think we could remove the docker logo, it is a bit too big for me.

Reply:
I think we should leave this image here because its highlights this section but we can decrease the image size if its too big.

Reply:
resized to 100 pixel witdth